testdata/p4_16_samples_outputs/pna-dpdk-header-stack-assignment.p4.spec

```diff
@@ -122,38 +122,38 @@ action encap_one_tunnel_layer_ipv4 args instanceof encap_one_tunnel_layer_ipv4_a
 	mov h.mac.type 0x800
 	validate h.ipv4_0
 	mov m.MainControlT_tmp h.ipv4_0.version_ihl
-	and m.MainControlT_tmp 0xF0
+	and m.MainControlT_tmp 0xF
```

Comment:
how come this bug wasn't caught?

Reply:
It was caught.  That is why there was a bug report about it.

Do you mean, why wasn't it caught earlier than Sep 2023 when I reported it?

Probably at least in part because there are no automated CI tests that pass data packets through the P4 DPDK that would have caught this.  Fabian Ruffy has tried to create some such tests, but he has had difficulty making them work consistently enough to use for CI purposes on this repository.

Reply:
And Perhaps no user tried accessing the individual header fields of the combined fields. Some of the early users may be combining these fields by themselves in the P4 program and thereby handling these extractions also in the program itself.
Thanks to Andy who caught this!!